process explorer/backend/ProcessExplorer/Processes/ProcessMonitor.cs

```diff
@@ -275,15 +291,24 @@ private async Task<bool> TryDeleteMainProcesses(int pid)
             return false;
         }
 
+        private void ModifyStatus(ProcessInfoDto item)
+        {
+            lock (locker)
+            {
+                var index = Data.Processes.IndexOf(item);
+                Data.Processes[index].ProcessStatus = Status.Terminated.ToStringCached();
+            }
+        }
+
         /// <summary>
         /// Delays the removing process from the list.
         /// </summary>
         /// <param name="item"></param>
-        private async void DeleteFromListAfter1Minute(ProcessInfoDto item)
+        private void DeleteFromListAfter1Minute(ProcessInfoDto item, int delay)
```

Comment:
I would remove the "1Minute" from the name here. Maybe "RemoveAfterTimeout".
It seems that the "DelayTime" is accessible from this method, so you don't need to pass it as an argument.